README.old.md

```diff
@@ -0,0 +1,54 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+Math Magicians
+> Description the project.
+
+
+## Built With
+
+- React
+- JSX
+- JavaScript
+
+## Live Demo (if available)
+
+[> coming soon]()
+
+
+## Getting Started
+run npm start in the terminal 
+
+
+## Authors
+
+## Authors
+
+👤 **Habeeb Musa Efiamotu**
+
+- GitHub: [@githubhandle](https://github.com/Efiamotu-1)
+- Twitter: [@twitterhandle](https://twitter.com/EFYAMOTU)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/musa-habeeb/)
+
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
+
+_NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish._
```

Comment:
- [ ] Please you give the wrong name to your readme file, I would like to recommend you edit the in **README.md** instead of **README.old.md** to make your readme file works as expected :+1: